src/Codec/Beam.hs

```diff
@@ -51,7 +51,7 @@ data Operand
   | Atom BS.ByteString
   | Reg Register
   | Lab Int
-  | ExtLiteral Encoding.Literal
+  | Ext Encoding.Literal
```

Comment:
I'm imagining that the `Encoding.Literal` param will be pulled out into an `External` type. Similarly to how we handle `Register`. That way we can keep the short names at the very top level (more BEAM-esque) and still capture the full language.